collection-renderer-lib/test/cmr/collection_renderer/test/services/collection_renderer.clj

```diff
@@ -25,27 +26,47 @@
 (deftest render-default-collection
   (testing "Get returned HTML"
     (let [coll expected-conversion/example-collection-record
-          ^String html (cr/render-collection (renderer-context) coll "C1234-PROV1")]
+          ^String html (cr/render-collection (renderer-context) coll "C1234-PROV1" {})]
```

Comment:
The reason I pass in the empty map is to avoid calling a search since there is no instance of search running it will cause an exception. Is there a better way to do this?

Reply:
I assumed that `{}` was the default if the parameter wasn't passed in. You can ignore my comment and leave this the way it is.